decidim-core/lib/decidim/social_share_service_manifest.rb

```diff
@@ -54,7 +54,7 @@ def link?
     end
 
     # Add optional parameters to a share_uri
-    # This is initially developed for Twitter, as they allow sending a Hashtag and Via as parameters
+    # This is initially developed for Twitter, as they allow sending a Via as parameters
```

Comment:
Same here, we didn't use this feature never as far as I know, so let's remove this method

Reply:
Removed and updated references calling method in the file.